ldk/javascript/examples/self-test-loop/src/tests/network/index.ts

```diff
@@ -155,25 +143,25 @@ export const testWebsocketConnection = (): Promise<boolean> =>
 
           clipboardListener.cancel();
           console.info('Clipboard listener cancelled');
-
           cancellable.cancel();
         });
+
+        await socket.setCloseHandler((error, code, text) => {
+          if (error) {
+            console.error('setCloseHandler error', error);
+            reject(error);
+          }
+
+          console.info(`Received on close code: ${code}. ${text}`);
+
+          // This is from us closing the socket connection in the message handler below
+          if (testPassed) {
+            resolve(true);
+          }
```

Comment:
why dont you just do something like 

```
if (message === 'You are using a test api key') {
  resolve(true);
}
```
and get rid of the testPassed?

Reply:
does setCloseHandler not have access to message? I thought it would, but if not just ignore me.